sphinx_gallery/gen_rst.py

```diff
@@ -498,7 +499,8 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf):
     ref_fname = example_file.replace(os.path.sep, '_')
     example_rst = """\n\n.. _sphx_glr_{0}:\n\n""".format(ref_fname)
 
-    if not fname.startswith('plot'):
+    fname_pattern = gallery_conf.get('fname_pattern', 'plot_*')
+    if os.path.join(src_dir, fname) not in glob.glob(fname_pattern):
```

Comment:
On Thu, Jan 7, 2016 at 10:55 PM, Loïc Estève <notifications@github.com>
wrote:

> In sphinx_gallery/gen_rst.py
> <https://github.com/sphinx-gallery/sphinx-gallery/pull/85#discussion_r49099520>
> :
>
> > @@ -498,7 +499,8 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf):
> >      ref_fname = example_file.replace(os.path.sep, '_')
> >      example_rst = """\n\n.. _sphx_glr_{0}:\n\n""".format(ref_fname)
> >
> > -    if not fname.startswith('plot'):
> > +    fname_pattern = gallery_conf.get('fname_pattern', 'plot_*')
> > +    if os.path.join(src_dir, fname) not in glob.glob(fname_pattern):
>
> I'd rather have re.search on the absolute path of the example with regex
> pattern as I mentioned in the associated issue. This way it is more
> flexible. I would argue that this is a reasonably advanced feature so it is
> not too much to expect the person using it to know about regex.
>
what do you envision as the default in your system?

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sphinx-gallery/sphinx-gallery/pull/85/files#r49099520>
> .
>